### DISCUSSION TITLE
How to prevent Freeplane from removing trailing blanks in node text ?

### DISCUSSION THREAD

**Discussion by joeUser555** [freeplane/freeplane#1155] [Ideas]
**Describe the bug**
I want to have trailing blank in node text.
I searched the preferences for a disabling option but did not find.

**To Reproduce**
1. Enter some text with a trailing blank
2. Press enter to leave the node text editor
3. The trailing blank gets removed by Freeplane

**Expected behavior**
An option to disable trailing blank removal.

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04
 - Version 1.10.6
 - Java Version 11.0.18

**Suggestion**
Check for other automatic input text correction/processing 
and implement a disabling option for each behavior.

**Comment by macmarrum** [comment]
I know of no such option.
As a workaround, when I need a trailing space, I use a formula, e.g. `='Some text with a trailing space '` or use a script to enter the text, e.g.
```groovy
node.text = 'Some text with a trailing space '
```
but this one is tricky, because once I edit the text, the trailing space is removed.

**Comment by dpolivaev** [comment]
I converted this issue into an idea. If you agree that leading spaces should not be removed I can change it,

**Reply by bbyykk** [reply]
I like this feature. Please add the edit option for users to decide if you want to change it
